XMLHttpRequest/send-redirect-to-cors.htm

```diff
@@ -4,65 +4,64 @@
     <title>XMLHttpRequest: send() - Redirect to CORS-enabled resource</title>
     <script src="/resources/testharness.js"></script>
     <script src="/resources/testharnessreport.js"></script>
-    <link rel="help" href="https://xhr.spec.whatwg.org/#infrastructure-for-the-send()-method" data-tested-assertations="following::dl[1]/dt[2] following::dl[1]/dd[2]/ol/li[1] following::dl[1]/dd[2]/ol/li[3]" />
   </head>
   <body>
     <div id="log"></div>
     <script>
       function extractBody(body) {
         if (body === null) {
           return { body: "", type: "NO" };
-        }
-
-        if (typeof body == "string") {
+        } else if (typeof body == "string") {
           return { body: body, type: "text/plain;charset=UTF-8" };
-        }
-
-        if (body instanceof Uint8Array) {
+        } else if (body instanceof Uint8Array) {
           var arr = Array.prototype.slice.call(body);
           return { body: String.fromCharCode.apply(null, arr), type: "NO" }
+        } else {
+          return { body: "EXTRACT NOT IMPLEMENTED", type: "EXTRACT NOT IMPLEMENTED" }
```

Comment:
Fwiw, the wrapping of the body/type bits wasn't because it was too wide; it was just because it's more readable (for me at least) to have the different props on different lines....  I don't feel super-strongly about it, though.